about.html

```diff
@@ -0,0 +1,179 @@
+        <li><a href="#"> Program </a></li>
+        <li><a href="#"> Join </a></li>
+        <li><a href="#"> Sponsor </a></li>
+        <li><a href="#"> News </a></li>
+        <li><a href="#" class="lastCCA"> CC Campaign </a></li>
+      </ul>
+    </nav>
+
+    <nav class="mobile-nav">
+      <ul>
+        <li class="close_btn">
+          <i class="fa-solid fa-xmark" onclick="hamburger()"></i>
+        </li>
+        <li><a href="about.html"> About </a></li>
+        <li><a href="#"> Program </a></li>
+        <li><a href="#"> Join </a></li>
+        <li><a href="#"> Sponsor </a></li>
+        <li><a href="#"> News </a></li>
+        <li><a href="#"> CC Campaign </a></li>
+      </ul>
+    </nav>
+    <header class="Landing_Page">
+      <div id="hamburger">
+        <i class="fa-solid fa-bars" onclick="hamburger()"></i>
+      </div>
+      <p class="hello">"Hello! Sharing World"</p>
+      <h1>
+        CREATIVE COMMONS <br />
+        GLOBAL SUMMIT <br />
+        2015
+      </h1>
+      <p class="message">
+        The Creative Commons Global Summit , held every two years, is a place
+        where CC activists from all over world and experts, scholars and
+        activists from all walks of life who support and practice open culture
+        gather together. In particular, this year's CC Summit awaits the
+        participation of various organizations and individual participants who
+        want to participate in activities to spread the value of sharing, free
+        culture, and open knowledge with CC. Free software supporters, Wikipedia
+        activists, open knowledge foundation activists, art galleries,
+        libraries, museums, archives, governments, foundations, lawyers, social
+        activists, etc. various people active in the field of open content and
+        open knowledge gather in one place to share richer and more dynamic We
+        hope that it will be an opportunity to share our thoughts on creating an
+        ecosystem.
+      </p>
+
+      <p class="place">
+        please contact us per email for any further questions about CC Global
+        Summit 2015!
+      </p>
+      <a class="date" href="mailto:skj997788@gmail.com"> Mail US </a>
+    </header>
+    <section>
+      <div class="wrap program-info-wraper">
+        <section>
+          <h2 class="normal">Creative Commons</h2>
+          <h2>Global Summit 2015 Logo</h2>
+          <div class="guide-bar"></div>
+          <div class="program-list">
+            <div class="program-card">
+              <p>
+                The logo of CC Global Summit was decided through the logo
+                competition from 8.jun to 7.july. After online public voting and
+                selection Committee judging process. the logo from Naresh
+                Agrawal won the prize.After consultations from Mr. Agrawal we
+                optimized the prize-wining-logo for thi syears summit as
+                following:
+              </p>
+            </div>
```

Comment:
- [ ] Please stick with Behance design here notice the border around the paragraph

Your desing 
![image](https://user-images.githubusercontent.com/74744787/163300274-c3d1e069-5667-47fc-9c39-36c10c6c12b2.png)

Behance design 

![image](https://user-images.githubusercontent.com/74744787/163300358-22c34d5c-0782-43d6-a13f-f22c72406dfc.png)